### PR TITLE
LOG-1342: Bump master to 5.2

### PR DESCRIPTION
**Pull request by ewolinetz**
Bumping master to 5.2 as part of branching for 5.1 for feature freeze

### CODE REVIEW

File: bundle/manifests/elasticsearch-operator.clusterserviceversion.yaml [hunk 1]
```diff
@@ -115,7 +115,7 @@ metadata:
     operatorframework.io/arch.amd64: supported
     operatorframework.io/arch.ppc64le: supported
     operatorframework.io/arch.s390x: supported
-  name: elasticsearch-operator.v5.1.0
+  name: elasticsearch-operator.v5.2.0
```
Comment (jcantrill) [inline bundle/manifests/elasticsearch-operator.clusterserviceversion.yaml:118]: @vimalk78 do we need to set ` olm.skipRange: '>=4.6.0-0 <5.1.0'` to 5.3.0 on the max end

Comment (ewolinetz) [reply bundle/manifests/elasticsearch-operator.clusterserviceversion.yaml:118]: we don't use anything from `bundle/` for CPaaS IIRC

**Comment by jcantrill** [comment]
I dont expect this to pass until https://github.com/openshift/release/pull/18309 merges

**Comment by jcantrill** [comment]
Manually merging this job uses the CLO src image to run its test which is a combination of CLO 5.2 and EO 5.1.  There is no way to fix env substitution without merging an EO 5.2 change.

**Pull request merged**